fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -323,7 +323,7 @@ createTableStatement
      ;
 
 columnDesc
-    : identifier type charsetName? KEY? aggDesc? (NULL | NOT NULL)? (defaultDesc | AUTO_INCREMENT)? comment?
+    : identifier type charsetName? KEY? aggDesc? (NULL | NOT NULL)? (defaultDesc | AUTO_INCREMENT)? USE_ZONE_MAP? comment?
```

Comment:
The zone map index Syntax should consistent with bloom filter index and bitmap index.

Reply:
`PROPERTIES("zone_map_columns" = "k1,k2");`
`ALTER TABLE table1 SET ("zone_map_columns" = "k1,k2,v1");`
So, like this?